addons/mail/static/src/models/message_reaction_group/message_reaction_group.js

```diff
@@ -0,0 +1,137 @@
+/** @odoo-module **/
+
+import { registerNewModel } from '@mail/model/model_core';
+import { attr, many2many, many2one } from '@mail/model/model_field';
+import { insertAndReplace } from '@mail/model/model_field_command';
+const { markEventHandled } = require('@mail/utils/utils');
```

Comment:
runbot doesn't seem to like this one ;)

Reply:
And not only this one, I'll adapt the tests.